public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ApplicationInsightsEditor/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { AzureMonitorQuery } from '../../types';
+import { Card, Field, InlineField, InlineFieldRow, Input } from '@grafana/ui';
+
+const ReadOnlyTimeGrain = ({
+  timeGrainCount,
+  timeGrainType,
+  timeGrainUnit,
+}: {
+  timeGrainCount: string;
+  timeGrainType: string;
+  timeGrainUnit: string;
+}) => {
+  const timeFields = timeGrainType === 'specific' ? ['specific', timeGrainCount, timeGrainUnit] : [timeGrainType];
+
+  return (
+    <InlineField label="Timegrain">
+      <>
+        {timeFields.map((timeField) => (
+          <Input value={timeField} disabled={true} onChange={() => {}} key={timeField} width={10} />
+        ))}
+      </>
+    </InlineField>
+  );
+};
+
+const ApplicationInsightsEditor = ({ query }: { query: AzureMonitorQuery }) => {
+  const groupBy = query.appInsights?.dimension || [];
+
+  return (
+    <div data-testid="azure-monitor-application-insights-query-editor">
+      <InlineFieldRow>
+        <InlineField label="Metric" disabled={true}>
```

Comment:
If we use the `import { Field } from '../Field';` component we'll get nice consistently spaced labels :)